.cycloid.yml

```diff
@@ -0,0 +1,37 @@
+# Configuration of Cycloid stack
+---
+version: '2'
+name: 'Compute'
+canonical: 'stack-compute'
+status: 'public'
+description: |
+  This stack allows to deploy a configure compute instance in one of the available providers.
+keywords:
+  - 'compute'
+  - 'provider:aws'
+  - 'ec2'
+author: 'Cycloid'
+image: 'https://raw.githubusercontent.com/marcoldp/k8s-stack/main/docs/k8s-logo.png'
```

Comment:
Maybe add or create an appropriated logo In the worse case use this one https://github.com/cycloid-community-catalog/stack-getting-started/blob/master/icon.png